.github/CODEOWNERS

```diff
@@ -7,23 +7,16 @@
 # be requested for review when someone opens a pull request.
 # order is alphabetical for easier maintenance.
 #
-# Forrest Xia (forrestxia)
-# Yian Zong (YianZong)
-# Yiming Bao (baoy1)
-# Tao He (taohe1012)
-# Peter Cao (P-Cao)
-# Don Khan (donatwork)
-# Harsha Yalamanchili (harshaatdell)
-# AnandatDell (anandrajak1)
+# Aaron Tye (atye)
+# Aly Nathoo (anathoodell)
+# Chiman Jain (chimanjain)
+# Evgeny Uglov (EvgenyUglov)
+# Harish H  (HarishH-DELL)
 # Nitesh Rewatkar (nitesh3108)
+# Oleksandr babiychuk (alexemc)
```

Comment:
Last name should start with a capital letter.

Reply:
Fixed.